types/schemas/prototypeApplication/enums/PropertyTypes.ts

```diff
@@ -0,0 +1,2894 @@
+  | CommercialStorageLandGeneral
+  | CommercialTransport
+  | CommercialTransportAir
+  | CommercialTransportAirAirfield
+  | CommercialTransportAirAirport
+  | CommercialTransportAirHelicopterStation
+  | CommercialTransportAirHeliport
+  | CommercialTransportAirInfrastructure
+  | CommercialTransportAirPassengerTerminal
+  | CommercialTransportBus
+  | CommercialTransportDock
+  | CommercialTransportDockFerryPassengers
+  | CommercialTransportDockFerryVehicles
+  | CommercialTransportDockGeneralBerth
+  | CommercialTransportDockPassenger
+  | CommercialTransportDockRefuelling
+  | CommercialTransportDockSlipway
+  | CommercialTransportDockTankerBerth
+  | CommercialTransportFreight
+  | CommercialTransportFreightAir
+  | CommercialTransportFreightContainer
+  | CommercialTransportFreightRail
+  | CommercialTransportFreightRoad
+  | CommercialTransportInfrastructure
+  | CommercialTransportInfrastructureAqueduct
+  | CommercialTransportInfrastructureLock
+  | CommercialTransportInfrastructureWeighing
+  | CommercialTransportInfrastructureWeir
+  | CommercialTransportMarina
+  | CommercialTransportMooring
+  | CommercialTransportOvernightLorryPark
+  | CommercialTransportParking
+  | CommercialTransportParkingCar
+  | CommercialTransportParkingCoach
+  | CommercialTransportParkingCommercialVehicle
+  | CommercialTransportParkingParkAndRide
+  | CommercialTransportRailAsset
+  | CommercialTransportStorage
+  | CommercialTransportStorageBoat
+  | CommercialTransportStorageBus
+  | CommercialTransportTerminal
+  | CommercialTransportTerminalBrain
+  | CommercialTransportTerminalBus
+  | CommercialTransportTerminalVehicularRail
+  | CommercialTransportTrack
+  | CommercialTransportTrackCable
+  | CommercialTransportTrackCliff
+  | CommercialTransportTrackMonorail
+  | CommercialUtility
+  | CommercialUtilityDam
+  | CommercialUtilityElectricity
+  | CommercialUtilityElectricityDistribution
+  | CommercialUtilityElectricityProduction
+  | CommercialUtilityElectricityWindFarm
+  | CommercialUtilityElectricityWindTurbine
+  | CommercialUtilityLandfill
+  | CommercialUtilityOilGas
+  | CommercialUtilityOilGasGasGovernor
+  | CommercialUtilityOilGasGasHolder
+  | CommercialUtilityOilGasOilTerminal
+  | CommercialUtilityOther
+  | CommercialUtilityOtherCableTerminal
+  | CommercialUtilityOtherObservatory
+  | CommercialUtilityOtherRadar
+  | CommercialUtilityOtherSatelliteEarth
+  | CommercialUtilityPublicPhoneBox
+  | CommercialUtilityPublicPhoneOther
+  | CommercialUtilitySubStation
+  | CommercialUtilityTelecoms
+  | CommercialUtilityTelecomsExchange
+  | CommercialUtilityTelecomsMast
+  | CommercialUtilityWasteManagement
+  | CommercialUtilityWater
+  | CommercialUtilityWaterPumpControl
+  | CommercialUtilityWaterPumpDistribution
+  | CommercialUtilityWaterQualityMonitoring
+  | CommercialUtilityWaterStorage
+  | CommercialUtilityWaterTreatment
+  | CommercialUtilityWaterTreatmentWaste
+  | CommercialUtilityWaterTreatmentWater
```

Comment:
Some pretty gnarly variable names here! I've gone for just concatenating the dot separated variables as - 
 - We can still group and list alphabetically
 - There's an easy pattern to follow for new values
 - If we simplified down, we risk clashes / issues when adding new values in future
 - They're just internal to this repo and won't be referenced outside this file

Reply:
Makes sense!